Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml

```diff
@@ -74,13 +74,22 @@
 		<put-attribute name="currentPage" value="home" />
 	</definition>
 
+   <!-- checks -->
+   <definition name="check" extends="base.definition.nonav">
+       <put-attribute name="content" value="/WEB-INF/views/checks.jsp" />
+	</definition>
 
-    <!-- checks -->
-    <definition name="check" extends="base.definition.nonav">
-        <put-attribute name="content" value="/WEB-INF/views/checks.jsp" />
+    <!-- result -->
+    <definition name ="status_transform" extends="base.definition.nonav">
+        <put-attribute name="content" value="/WEB-INF/views/status_transform.jsp" />
```

Comment:
and the corresponding jsp files should be in the folder for transformation files